integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/util/SecondaryIndexUtil.scala

```diff
@@ -345,6 +353,14 @@ object SecondaryIndexUtil {
     }
   }
 
+
+  private def deleteOldCarbonDataFiles(partition: CarbonSparkPartition): Unit = {
```

Comment:
can u please add a method level comment why this method is used, basically the scenario

Reply:
done